src/arm64/implementation.cpp

```diff
@@ -1310,6 +1310,13 @@ size_t implementation::binary_to_base64(const char *input, size_t length,
                                         base64_options options) const noexcept {
   return encode_base64(output, input, length, options);
 }
+  #if SIMDUTF_ATOMIC_REF
+void implementation::memcpy_atomic_read(char *const dst, const char *const src,
+                                        const std::size_t len) const noexcept {
+  scalar::memcpy_atomic_read(dst, src, len);
```

Comment:
64-bit ARM is atomic when loading and storing 64-bit words.

<img width="820" alt="Screenshot 2025-04-28 at 10 13 45 AM" src="https://github.com/user-attachments/assets/83a78deb-0701-4f5b-9f77-c4e8908cc2ee" />